src/WorkflowStep.spec.ts

```diff
@@ -291,12 +291,12 @@ describe('WorkflowStep', () => {
 
   describe('processStepMiddleware', () => {
     it('should call each callback in user-provided middleware', async () => {
-      const { next: _next, ...fakeArgs } = createFakeStepEditAction() as unknown as AllWorkflowStepMiddlewareArgs;
```

Comment:
`createFakeStepEditAction()` (and others similar) adds `next` inside but we remove it from `fakeArgs` in this line. `AllWorkflowStepMiddlewareArgs` minus `next` property does not compile with `processStepMiddleware` in this test method. This is why I've removed the `next: sinon.fake()` in this test. Just in case, we can need to do more tests to verify that we are not breaking anything.

Reply:
I only vaguely remember writing this, but what I think I recall is that the removal of `next` here was to mimic the switch out that occurs of the default/usual `next` and the replacement of it with the user-provided callback. It's been a while, however, so not sure if this file has since been changed since that was originally written.